python/tests/test_bend_flux.py

```diff
@@ -118,6 +118,7 @@ def main(args):
 
     bend_flux(True)
     bend_flux(False)
+    mp.all_wait()
```

Comment:
All of these `all_wait` calls should not be necessary—Meep functions (like flux computation) do synchronization exactly where it is needed), and in general aren't a style I would recommend (for the most part, Meep users shouldn't need to care whether their scripts run in serial or in parallel).